integration-tests/src/test/resources/integration-test/buildIT/pom.xml

```diff
@@ -9,4 +9,13 @@
     <artifactId>owa</artifactId>
     <version>1.4</version>
     <packaging>pom</packaging>
+
+    <repositories>
+        <repository>
+            <id>openmrs-repo</id>
+            <name>OpenMRS repository</name>
+            <url>http://mavenrepo.openmrs.org/nexus/content/repositories/public</url>
+        </repository>
+    </repositories>
+
 </project>
```

Comment:
Note: I had to add the repositories section into 2 of the poms in the project to get the integration tests to run.  This is likely due to my having removed my settings.xml file from my .m2 directory recently.  This allows the unit tests to pass without this environmental dependency.